toolboxTreppe/toolboxTreppe.ino

```diff
@@ -116,9 +116,6 @@ void animation1() {
     leds[ XY(xB, m_y)] = CHSV( hue, 255, 255 );
     leds[ XY(5, gegenstrecke)] = CHSV( hue, 255, 255 );
     FastLED.show();
-    if (hue > 255) {
-      hue = 0;
```

Comment:
hue wird in 115 erhöht. reset nötig

Reply:
hue ist eine uint8_t das heißt nach 255 kommt 0 
weiterhin kann hue niemals größer als 255 sein und dementsprechend der check nie etwas machen

Reply:
btw bezüglich 115 weißt du zufällig ob das post increment da dann schon den erhöhten Wert weitergibt? ich dachte das ist dann erst für die nächste Funktion der erhöhte Wert (wobei ich natürlich auch nicht sicher weiß was da an der Stelle das Ziel ist daher habe ich das auch mal so gelassen)

Reply:
hast recht uint8 überlauf ist mir entgangen.
post inc macht wie der name schon sagt das inc hinterher bzw returnt den wert vor dem inc. vermutlich soll es hier ein pre sein

Reply:
ok dachte ich mir auch ;)

Reply:
vermutlich wäre es einfach sinnvoller das Inkrement dann aber davor oder danach zu stellen.
Das in der Übergabe zu machen ist eher unübersichtlich (und mit pre/post inkrement tendenziell auch fehleranfälliger)